01_Core/week04/ecommerce/contact/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width">
+		<title>Stardust Designs || Contemporary Furniture</title>
+		<link rel="stylesheet" href="../stylesheets/normalize.css">
+		<link rel="stylesheet" href="../stylesheets/style.css">
+		<script src="validation.js"></script>
+	</head><!--/head-->
+	<body>
+		<header>
+			<nav class="main-nav">
+				<ul class="nav__ul h-marg-0">
+					<li><img src="../images/logo_banner.png" class="logo"></li>
```

Comment:
From a semantic perspective, the image tag is problematic here. It's not flat-out wrong, but it's not really part of the list of links. CSS might work better. Regardless, the alt property is required on the img tag.